ads/google/test/test-adsense.js

```diff
@@ -71,4 +71,12 @@ describes.realWin('adsenseDelayedFetch', {}, env => {
       hid: pageViewId,
     });
   });
+
+  it('throws on invalid responsive ad unit height', () => {
+    const data = {'autoFormat': 'rspv', 'height': '666'};
+    expect(() => {
```

Comment:
No need for brackets, this can be:  expect(() => adsense(env.win, data)).to.throw(...)